src/FakeItEasy.Analyzer.CSharp/FakeItEasy.Analyzer.CSharp.csproj

```diff
@@ -6,7 +6,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <FileAlignment>512</FileAlignment>
     <SignAssembly>true</SignAssembly>
-    <DefineConstants>$(DefineConstants);CSHARP</DefineConstants>
+    <DefineConstants>$(DefineConstants);CSHARP;LACKS_NULLABLE_ATTRIBUTES</DefineConstants>

```

Comment:
Is this necessary? We don't need the `#if`s, since there's a single TFM... Or is it future-proofing if we target netstandard2.1 someday?

Reply:
Ah, I understand. The included file has `#if`s anyway, so we need the symbol